### PR TITLE
Ds 168 upgrade angular 12 2 1

### PR DESCRIPTION
**Pull request by famareme**

### CODE REVIEW

File: build-storybook.log [hunk 1]
```diff
@@ -644,7 +644,7 @@ Operating system : macOS Catalina
 [07:43:54]                : - declarations:
```
[the remark this answers is not in the document]
Comment (famareme) [reply build-storybook.log:644]: oui c un oublie; je vais l'enlever

File: projects/front-end-library/src/_doc/1-Introduction/1-introduction.stories.mdx [hunk 2]
```diff
@@ -1,19 +1,19 @@
-import { Meta } from '@storybook/addon-docs/blocks';
```
[the remark this answers is not in the document]
Comment (famareme) [reply projects/front-end-library/src/_doc/1-Introduction/1-introduction.stories.mdx:1]: oui le merge est déjà fait avec master

File: projects/front-end-library/src/lib/js/demo/bifrost/carousel.js [hunk 3]
```diff
@@ -13,18 +14,18 @@ bfSlick.assignCarousel('.js-bf-carousel-demo', {
...
-        settings: {
-            slidesToShow: 3,
-            slidesToScroll: 3
-        }
+        breakpoint: breakpoints.lg,
+            settings: {
+                slidesToShow: 3,
+                slidesToScroll: 3
+            }
         },
```
Comment (JonathanLefebvreV) [inline projects/front-end-library/src/lib/js/demo/bifrost/carousel.js:29]: L'indentation est pas égal

**Review by JonathanLefebvreV — CHANGES_REQUESTED** [review]

File: scripts/generate/templates/component/_doc/doc.stories.mdx [hunk 4]
```diff
@@ -1,11 +1,11 @@
 import { moduleMetadata }               from '@storybook/angular';
-import { Meta, Story, Preview, Props }  from '@storybook/addon-docs/blocks';
+import { Canvas,Meta, Story, Preview, Props }  from '@storybook/addon-docs';
```
Comment (JonathanLefebvreV) [inline scripts/generate/templates/component/_doc/doc.stories.mdx:2]:
```suggestion
import { Canvas, Meta, Story, Preview, Props }  from '@storybook/addon-docs';
```

**Review by JonathanLefebvreV — CHANGES_REQUESTED** [review]
Actual:
image
https://bifrost-front-end-library-git-ds-168-upgrade-404634-bifrost-vui.vercel.app/?path=/story/components-carousel--drupal

Expected: 
image
https://bifrost-front-end-library-vui.vercel.app/?path=/story/components-carousel--drupal&globals=backgrounds.grid:false

**Review by JonathanLefebvreV — CHANGES_REQUESTED** [review]
input increment doesn't work on your branch
https://bifrost-front-end-library-git-ds-168-upgrade-404634-bifrost-vui.vercel.app/?path=/story/components-input-increment--drupal
Erreur de Mime type coté server:
image
En local le carrousel et le input-increment fonctionnent:
image


image

**Review by JonathanLefebvreV — CHANGES_REQUESTED** [review]
Slide show doesn't work.
https://bifrost-front-end-library-git-ds-168-upgrade-404634-bifrost-vui.vercel.app/?path=/story/components-slideshow--drupal

Which is probably the same problem then Carousel as they used the same library.
Seeing the input increment failed too, the problem seem related to all JS file load

Le script vendors~bifrost-components.js n'est pas généré coté serveur:
image

**Review by JonathanLefebvreV — APPROVED** [review]

**Pull request merged**